manifests/test/percona-server-mongodb.yaml

```diff
@@ -0,0 +1,334 @@
+apiVersion: psmdb.percona.com/v1-6-0
```

Comment:
Can we make this file as minimal as possible ?

Reply:
Sure. I was on the fence between make it minimal or keep the original CR pulling from their repo so we can reference it later (act as a document) but I guess we can always go to their repo/website to read up about all options.